pattoo/api/portal/react/src/AgentsHead.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class AgentsHead extends React.Component {
+    render() {
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.